src/utils/hours.js

```diff
@@ -1,4 +1,4 @@
-import * as moment from 'moment';
+import { isSameDay, parseISO, getDay, parse, format } from 'date-fns';
```

Comment:
```suggestion
import { parseISO, getDay, parse, format } from 'date-fns';
```
`isSameDay` is not needed if using the simplified code below.